quickstack/manifests/compute_common.pp

```diff
@@ -426,18 +426,44 @@
 
   class { 'filebeat':
     outputs => {
-      'logstash'  => {
-        'hosts'       =>  [$elasticsearch_host],
+      'logstash' => {
+        'hosts'       => [$elasticsearch_host],
         'loadbalance' => true
       }
     },
     logging => {
-      'level' => "info"
+      'level'    => "info"
```

Comment:
I think this line and the similar line in controller_common.pp are OK without spaces actually, it is inside its own {}.  But, you can fix it next time.